lib/browser.ts

```diff
@@ -978,16 +952,14 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
         .then(() => {
           // Reset bpClient sync
           if (this.bpClient) {
-            return this.driver.controlFlow().execute(() => {
-              return this.bpClient.setWaitEnabled(!this.internalIgnoreSynchronization);
-            });
+            return this.bpClient.setWaitEnabled(!this.internalIgnoreSynchronization);
           }
         })
         .then(() => {
```

Comment:
This promise chain should eventually become async/await, but let's save that for another Pr.

Reply:
I agree, this promise chain might be better contained in a separate PR.